src/isomorphic-fetch.js

```diff
@@ -1,9 +1,13 @@
+/* globals require */
+
 import global from './metal/global';
 import isNodeLikeEnvironment from './metal/is-node-like-environment';
 
 const fetch = global.fetch;
 
 if (!fetch && isNodeLikeEnvironment()) {
-  global.fetch = global.require('node-fetch');
+  const localRequire = require;
```

Comment:
Can you verify that this indirection is sufficient to pass `npm run build`? I would also annotate the reason for the indirection.